src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1916,7 +1916,7 @@ private function registerMailerConfiguration(array $config, ContainerBuilder $co
         }
 
         $loader->load('mailer.xml');
-        $container->getDefinition('mailer.transport')->setArgument(0, $config['dsn']);
+        $container->getDefinition('mailer.default_transport')->setArgument(0, $config['dsn']);
```

Comment:
`mailer.transport` would conflict with https://github.com/symfony/monolog-bundle/blob/master/DependencyInjection/Compiler/AddSwiftMailerTransportPass.php#L40 if `$mailerId` is `mailer`